runtime/gc_glue_java/EnvironmentDelegate.hpp

```diff
@@ -28,9 +28,17 @@
 
 #include "MarkJavaStats.hpp"
 #include "ScavengerJavaStats.hpp"
+#include "GCExtensionsBase.hpp"
+#include "ObjectModel.hpp"
 
 struct OMR_VMThread;
 
+typedef struct GCmovedObjectHashCode {
+	uint32_t originalHashCode;
+	BOOLEAN hasBeenMoved;
+	BOOLEAN hasBeenHashed;
```

Comment:
I'd use C++ style 'bool'.

I personally don't like using C style typedef struct either, but it's widespread already, so you can leave it.